server/controllers/recommendations/dateValidation/formValidator.ts

```diff
@@ -0,0 +1,52 @@
+import { formatValidationErrorMessage, invalidDateInputPart, makeErrorObject } from '../../../utils/errors'
+import { dateHasError } from '../../../utils/dates'
+import { ValidationError } from '../../../@types/dates'
+import { convertGmtDatePartsToUtc } from '../../../utils/dates/convert'
+import { FormValidatorArgs, FormValidatorReturn } from '../../../@types/pagesForms'
+
+export const validateDateTime = async ({ requestBody }: FormValidatorArgs): FormValidatorReturn => {
+  let errors
+
+  const dateTimeParts = {
+    day: requestBody['dateTime-day'],
+    month: requestBody['dateTime-month'],
+    year: requestBody['dateTime-year'],
+    hour: requestBody['dateTime-hour'],
+    minute: requestBody['dateTime-minute'],
+  }
+  const dateTimeIso = convertGmtDatePartsToUtc(dateTimeParts as Record<string, string>, {
+    includeTime: true,
+    dateMustBeInFuture: false,
+    dateMustBeInPast: false,
+    validatePartLengths: false,
+  })
+
+  if (dateHasError(dateTimeIso)) {
+    errors = []
+    if (dateHasError(dateTimeIso)) {
+      errors.push(
+        makeErrorObject({
+          name: 'dateTime',
+          id: invalidDateInputPart(dateTimeIso as ValidationError, 'dateTime'),
+          text: formatValidationErrorMessage(dateTimeIso as ValidationError, 'date the SPO agreed the recall'),
```

Comment:
'date the SPO agreed the recall' seems like an error that is too specific in something that looks like it is intended to be generic.

Reply:
I have changed the folder from `dateValidation` to `validateSpoDecisionDate`